src/docker-compose.override.yml

```diff
@@ -0,0 +1,8 @@
+version: '3.4'
+
+services:
+  blazorboilerplate:
+    environment:
+      - ASPNETCORE_ENVIRONMENT=Development
+    ports:
+      - "80"
```

Comment:
whole file is not needed, it does not override any settings from the main docker-compose.yml

What I'd suggest, is to move the environment/ports section from main file into override file.
That way, in the main docker-compose.yml we have a declaration of services/images/build and in the 'runtime' file like.override.yml we have the runtime (contaiers) specific settings like environment variables values/ports etc